hack/test-integration.sh

```diff
@@ -60,5 +60,8 @@ export childargs
 
 # run each test as its own process
 pushd "./${package}" 2>&1 >/dev/null
-time go run "${OS_ROOT}/hack/listtests.go" -prefix="${OS_GO_PACKAGE}/${package}.Test" "${testdir}" | xargs -I {} -n 1 bash -c "exectest {} ${@:1}" # "${testexec}" -test.run="^{}$" "${@:1}"
+# first run just lists the tests and ensures the list is non-zero length.  listtests will return non-zero
+# if no tests are found.
+go run "${OS_ROOT}/hack/listtests.go" -prefix="${OS_GO_PACKAGE}/${package}.Test" "${testdir}" > /dev/null
```

Comment:
it didn't when i tried returning non-zero from listgo, but maybe log.Fatalf does something more severe? i'll test it.